.circleci/config.yml

```
@@ -109,5 +109,7 @@ workflows:
             - lint
             - test
           filters:
+            branches:
+              ignore: .*
```

```suggestion
              ignore: /.*/
```
Regex must be enclosed in slashes... Read the docs that you intend to quote!